inc/console/migration/racksplugintocorecommand.class.php

```diff
@@ -546,11 +546,11 @@ private function importOtherElements() {
                new ChoiceQuestion(
                   sprintf(__('Where do you want to import "%s" ?'), $model_label),
                   [
-                     self::OTHER_TYPE_CHOICE_COMPUTER            => __('Computer'),
-                     self::OTHER_TYPE_CHOICE_NETWORKEQUIPEMENT   => __('Network device'),
+                     self::OTHER_TYPE_CHOICE_COMPUTER            => Computer::getTypeName(1),
+                     self::OTHER_TYPE_CHOICE_NETWORKEQUIPEMENT   => NetworkEquipment::getTypeName(1),
                      self::OTHER_TYPE_CHOICE_PERIPHERAL          => __('Peripheral'),
```

Comment:
why not doing the 3 others ?

Reply:
Because they were not part of abigous ones :D

Reply:
Voila; it's done